### ISSUE TITLE
Cannot build or run tests with Bazel 0.4.4

### ISSUE THREAD

**Issue by prestonvanloon** [pubref/rules_protobuf#80]
```bash
bazel test //...:all # bazel build //...:all does the same 
ERROR: /private/tmp/rules_protobuf/WORKSPACE:113:1: Cannot redefine repository after any load statement in the WORKSPACE file (for repository 'org_pubref_rules_protobuf').
ERROR: Error evaluating WORKSPACE file.
ERROR: package contains errors: objc.
ERROR: error loading package 'external': Package 'external' contains errors.
INFO: Elapsed time: 6.806s
ERROR: Couldn't start the build. Unable to run tests.
```

```bash
bazel version
Build label: 0.4.4-homebrew
Build target: bazel-out/local-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Feb 2 01:05:15 2017 (1485997515)
Build timestamp: 1485997515
Build timestamp as int: 1485997515
```

Note: 0.4.5 is latest.

**Comment by prestonvanloon** [comment]
I think this might be a duplicate of #67, but I did not see a fix committed.

**Comment by pcj** [comment]
Yeah I need to get rid of that!  Yeesh. I actually was working on that this morning and have a fix for the tests/external_proto... But it uncovered one other issue.  Plan to get a fix for #67 and #79 later this evening.

**Comment by prestonvanloon** [comment]
I'll see if I can send you a PR, if you haven't fixed it yet.

**Comment by prestonvanloon** [comment]
I made some progress but running into a weird issue. https://github.com/prestonvanloon/rules_protobuf/blob/master/tests/external_proto_library/BUILD#L53

Not sure if this is the right approach. Let me know.

**Comment by pcj** [comment]
Yep that's where I was going this AM as well, almost same workspace with googleapis.

The issue is in the go_proto_compile and cc_proto_compile whereby the output directory for either the files / descriptor set does not exist.  Protoc does not like that.  So there needs to be an extra 'mkdir' command inserted into the command list at https://github.com/prestonvanloon/rules_protobuf/blob/master/protobuf/internal/proto_compile.bzl#L372. I think I have a branch where this functionality exists (thought I fixed it already), so I will hunt that down.

**Comment by pcj** [comment]
Took awhile but should be fixed by #82
